cqrs-todoql-psql/src/graphql/schema.rs

```diff
@@ -166,6 +198,96 @@ graphql_object!(<'a> PageInfo<'a>: Context as "PageInfo" |&self| {
     }
 });
 
+#[derive(Clone, Debug)]
+struct VersionedTodoEventQL(cqrs::VersionedEventWithMetadata<TodoEvent, TodoMetadata>);
+
+graphql_object!(VersionedTodoEventQL: Context as "VersionedTodoEvent" |&self| {
+    description: "A versioned event in the history of a todo aggregate"
+
+    /// The event
+    field event() -> TodoEventQL {
+        TodoEventQL(&self.0.event)
+    }
+
+    /// The sequence number of the event in the event stream of the aggregate
+    field sequence() -> i32 {
+        self.0.sequence.get().to_i32().unwrap_or_else(i32::max_value)
```

Comment:
Does this hide any issues where the sequence number ever gets larger than `i32`?

edit: I just noticed this is in the `todo` example code, so this might be fine

Reply:
Yeah, it's a bit of a limitation in the GraphQL API that the maximum value is i32. We could instead represent event numbers as strings, and that would allow us to have arbitrary precision. This applies not only here, but in our own internal use.